R/plotLoadings.R

```diff
@@ -0,0 +1,385 @@
+#' Plot feature loadings after performing a reducedDim 
+#'
+#' Inspired by the \code{\link[diffTop:plotASVcircular]{plotASVcircular}} method using phyloseq 
+#' and has been converted to use TreeSummarizedExperiment objects.
+#'
+#' @param x a
+#'   \code{\link[TreeSummarizedExperiment:TreeSummarizedExperiment-class]{TreeSummarizedExperiment}}
+#'   x.
+#' 
+#' @param dimred \code{Character scalar}. Dimension reduction from reducedDimNames to be plotted. 
+#'   (default: \code{"PCA"})
+#'  
+#' @param layout One way to plot feature loadings of \code{c("heatmap", "barplot", "tree")} 
+#'   (default: \code{"heatmap"})
+#' 
+#' @param n A numeric specifying the number of features to be plotted.
+#'   (default: \code{10})
+#'   
+#' @param ncomponents A numeric specifying the number of components.
+#'   (default: \code{5})
+#' 
+#' @param tree.name A single \code{character} value specifying a rowTree from a
+#'   TreeSummarizedExperiment object. (By default: \code{"phylo"})
+#'   
+#' @param class A single \code{character} value specifying a rank from taxonomyRanks
+#'   (default: \code{rownames})
+#'     
+#' 
+#' @details
```

Comment:
You can also say here something about the limitations related to feature and sample set sizes etc.